Module not found error

**Describe the bug**
I received module not found ERROR for all the components from Kitchen sink example. I am running it in windows instance.


**To Reproduce**
Steps to reproduce the behavior:
1) DOwnloaded the project zip from github
2) EXtracted the aem-master
3) RAn yarn from the root and then yarn build
4) THen opened a new terminal from kitchen-sink example and ran the command yarn storybook
5) ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/breadcrumb/v2/breadcrumb/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentreadcrumbreadcrumb/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\breadcrumb\v2\breadcrumb'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/breadcrumb/v2/breadcrumb/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/breadcrumb/v2/breadcrumb/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentreadcrumbreadcrumb/breadcrumb.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\breadcrumb\v2\breadcrumb'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/breadcrumb/v2/breadcrumb/.content.xml 14:19-148
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/button/v1/button/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentuttonutton/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\button\v1\button'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/button/v1/button/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/button/v1/button/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentuttonutton/button.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\button\v1\button'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/button/v1/button/.content.xml 14:19-136
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/tabs/v1/tabs/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   abs1  abs/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\tabs\v1\tabs'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/tabs/v1/tabs/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/tabs/v1/tabs/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   abs1  abs/tabs.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\tabs\v1\tabs'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/tabs/v1/tabs/.content.xml 14:19-130
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/teaser/v1/teaser/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   easer1
easer/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\teaser\v1\teaser'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/teaser/v1/teaser/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/teaser/v1/teaser/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   easer1
easer/teaser.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\teaser\v1\teaser'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/teaser/v1/teaser/.content.xml 14:19-136
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/text/v2/text/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   ext2  ext/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\text\v2\text'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/text/v2/text/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/text/v2/text/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   ext2  ext/text.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\text\v2\text'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/text/v2/text/.content.xml 14:19-130
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/title/v2/title/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   itle2 itle/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\title\v2\title'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/title/v2/title/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/title/v2/title/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponents   itle2 itle/title.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\title\v2\title'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/title/v2/title/.content.xml 14:19-133
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/button/v2/button/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsoruttonutton/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\form\button\v2\button'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/button/v2/button/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/button/v2/button/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsoruttonutton/button.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\form\button\v2\button'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/button/v2/button/.content.xml 14:19-141
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/text/v2/text/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsorm      ext2    ext/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\form\text\v2\text'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/text/v2/text/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/text/v2/text/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsorm      ext2    ext/text.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\form\text\v2\text'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/form/text/v2/text/.content.xml 14:19-135
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/accordion/v1/accordion/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsaccordion1accordion/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\accordion\v1\accordion'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/accordion/v1/accordion/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/accordion/v1/accordion/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsaccordion1accordion/accordion.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\accordion\v1\accordion'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/accordion/v1/accordion/.content.xml 14:19-145
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/image/v2/image/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsimage2image/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\image\v2\image'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/image/v2/image/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/image/v2/image/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsimage2image/image.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\image\v2\image'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/image/v2/image/.content.xml 14:19-133
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/list/v2/list/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentslist2list/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\list\v2\list'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/list/v2/list/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/list/v2/list/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentslist2list/list.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\list\v2\list'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/list/v2/list/.content.xml 14:19-130
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/progressbar/v1/progressbar/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsprogressba1progressbar/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\progressbar\v1\progressbar'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/progressbar/v1/progressbar/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/progressbar/v1/progressbar/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsprogressba1progressbar/progressbar.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\progressbar\v1\progressbar'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/progressbar/v1/progressbar/.content.xml 14:19-151
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/separator/v1/separator/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsseparator1separator/' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\separator\v1\separator'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/separator/v1/separator/.content.xml 4:29-10:5
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/separator/v1/separator/.content.xml
Module not found: Error: Can't resolve 'G:aem-masteraem-sb-js-core-componentssrcjcr_rootappscorewcmcomponentsseparator1separator/separator.html' in 'G:\aem-master\aem-sb-js-core-components\src\jcr_root\apps\core\wcm\components\separator\v1\separator'
 @ G:/aem-master/aem-sb-js-core-components/src/jcr_root/apps/core/wcm/components/separator/v1/separator/.content.xml 14:19-145
 @ G:/aem-master/aem-sb-js-core-components/components.js
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/text/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponents  ext/' in 'G:\aem-master\examples\aem-kitchen-sink\components\text'
 @ ./components/text/.content.xml 4:29-10:5
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/text/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponents  ext/text.html' in 'G:\aem-master\examples\aem-kitchen-sink\components\text'
 @ ./components/text/.content.xml 14:19-95
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/accordion/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponentsaccordion/' in 'G:\aem-master\examples\aem-kitchen-sink\components\accordion'
 @ ./components/accordion/.content.xml 4:29-10:5
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/accordion/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponentsaccordion/accordion.html' in 'G:\aem-master\examples\aem-kitchen-sink\components\accordion'
 @ ./components/accordion/.content.xml 14:19-105
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/aemtext/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponentsaemtext/' in 'G:\aem-master\examples\aem-kitchen-sink\components\aemtext'
 @ ./components/aemtext/.content.xml 4:29-10:5
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/list/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponentslist/' in 'G:\aem-master\examples\aem-kitchen-sink\components\list'
 @ ./components/list/.content.xml 4:29-10:5
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in ./components/list/.content.xml
Module not found: Error: Can't resolve 'G:aem-masterexamplesaem-kitchen-sinkcomponentslist/list.html' in 'G:\aem-master\examples\aem-kitchen-sink\components\list'
 @ ./components/list/.content.xml 14:19-95
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/List.js 10:15
Module parse failed: Unexpected token (10:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class List  {
>   static clazz = 'com.adobe.cq.wcm.core.components.models.List';
|
|   constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 12:50-111
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Component.js 14:15
Module parse failed: Unexpected token (14:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|    * @since com.adobe.cq.wcm.core.components.models 12.8.0
|    */
>   static PN_ID = "id";
|
|   /**
 @ G:/aem-master/aem-sb-js-core-components/models.js 7:55-121
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Breadcrumb.js 15:24
Module parse failed: Unexpected token (15:24)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|    * @since com.adobe.cq.wcm.core.components.models 11.1.0
|    */
>   static PN_SHOW_HIDDEN = "showHidden";
|
|   /**
 @ G:/aem-master/aem-sb-js-core-components/models.js 4:56-123
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Teaser.js 15:30
Module parse failed: Unexpected token (15:30)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|      * @since com.adobe.cq.wcm.core.components.models 12.4.0
|      */
>     static PN_ACTIONS_ENABLED = "actionsEnabled";
|
|     /**
 @ G:/aem-master/aem-sb-js-core-components/models.js 11:52-115
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Container.js 18:37
Module parse failed: Unexpected token (18:37)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|    * @since com.adobe.cq.wcm.core.components.models 12.8.0
|    */
>   static PN_BACKGROUND_IMAGE_ENABLED = "backgroundImageEnabled";
|
|   /**
 @ G:/aem-master/aem-sb-js-core-components/models.js 6:55-121
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Image.js 18:44
Module parse failed: Unexpected token (18:44)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|    * @since com.adobe.cq.wcm.core.components.models 11.0.0
|    */
>   static PN_DESIGN_ALLOWED_RENDITION_WIDTHS = "allowedRenditionWidths";
|
|   /**
 @ G:/aem-master/aem-sb-js-core-components/models.js 8:51-113
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Button.js 2:17
Module parse failed: Unexpected token (2:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default class Button {
>     static clazz = 'com.adobe.cq.wcm.core.components.models.Button';
|
|     constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 5:52-115
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/form/Button.js 2:17
Module parse failed: Unexpected token (2:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default class Button {
>     static clazz = 'com.adobe.cq.wcm.core.components.models.form.Button';
|
|     constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 16:57-125
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/form/Text.js 2:17
Module parse failed: Unexpected token (2:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default class Text {
>     static clazz = 'com.adobe.cq.wcm.core.components.models.form.Text';
|
|     constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 17:55-121
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/ListItem.js 7:15
Module parse failed: Unexpected token (7:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class ListItem {
>   static clazz = 'com.adobe.cq.wcm.core.components.models.ListItem';
|
|   constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 13:54-119
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Text.js 7:15
Module parse failed: Unexpected token (7:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class Text {
>   static clazz = 'com.adobe.cq.wcm.core.components.models.Text';
|
|   constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 9:50-111
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Title.js 7:15
Module parse failed: Unexpected token (7:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class Title {
>   static clazz = 'com.adobe.cq.wcm.core.components.models.Title';
|
|   constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 10:51-113
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Separator.js 7:17
Module parse failed: Unexpected token (7:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class Separator {
>     static clazz = 'com.adobe.cq.wcm.core.components.models.Separator';
|
|     constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 15:55-121
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/ProgressBar.js 7:17
Module parse failed: Unexpected token (7:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class Text {
>     static clazz = 'com.adobe.cq.wcm.core.components.models.ProgressBar';
|
|     constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 14:57-125
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Accordion.js 9:15
Module parse failed: Unexpected token (9:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class Accordion extends Container {
>   static clazz = 'com.adobe.cq.wcm.core.components.models.Accordion';
|
|   constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 3:55-121
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

ERROR in G:/aem-master/aem-sb-js-core-components/src/com/adobe/cq/wcm/core/components/models/Tabs.js 9:15
Module parse failed: Unexpected token (9:15)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| export default class Tabs extends PanelContainer {
>   static clazz = 'com.adobe.cq.wcm.core.components.models.Tabs';
|
|   constructor(content) {
 @ G:/aem-master/aem-sb-js-core-components/models.js 18:50-111
 @ G:/aem-master/aem-sb-js-core-components/index.js
 @ ./.storybook/preview.js
 @ multi G:/aem-master/node_modules/@storybook/core/dist/server/common/polyfills.js G:/aem-master/node_modules/@storybook/core/dist/server/preview/globals.js G:/aem-master/node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./.storybook/preview.js G:/aem-master/node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false.............................


AND it goes on

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Code snippets**
If applicable, add code samples to help explain your problem.

**System:**
Please paste the results of `npx -p @storybook/cli@next sb info` here.

**Additional context**
Add any other context about the problem here.

--- a/app/aem/src/server/aem-clientlib-txt-loader.js
+++ b/app/aem/src/server/aem-clientlib-txt-loader.js
@@ -2,19 +2,16 @@ import { sep as pathSeparator } from 'path';
 
 const NEW_LINE = '\n';
 
+const extractBasePath = basePathExpression => {
+  return basePathExpression.replace(/^\s*#base\s*=\s*/, '').replace(/\r/, '');
+};
 
-function extractBasePath(basePathExpression) {
-  return basePathExpression.replace(/^\s*#base\s*=\s*/, '');
-}
-
-function resolveDependencyFolder(contextPath, source) {
+const resolveDependencyFolder = (contextPath, source) => {
   const baseExpression = source.split(NEW_LINE).find(line => /^\s*#\s*base/.test(line));
   return contextPath + (baseExpression ? `/${extractBasePath(baseExpression)}` : '');
-}
+};
 
-function isAbsolutePath(path) {
-  return path.charAt(0) === '/';
-}
+const isAbsolutePath = filepath => filepath.charAt(0) === '/';
 
 export default function aemClientLibTxtLoader(source) {
   const depedencies = source
@@ -24,13 +21,19 @@ export default function aemClientLibTxtLoader(source) {
     // Ignore empty rows
     .filter(line => line);
 
-  const dependencyFolder = resolveDependencyFolder(this.context.split(pathSeparator).join('/'), source);
-  const requireCalls = depedencies.map(dependency => {
-    const trimmedDependency = isAbsolutePath(dependency) ? dependency.trim() : `/${dependency.trim()}`;
-    const filepath = `${dependencyFolder}${trimmedDependency}`;
-    console.log('\n[aemClientLibTxtLoader] filepath:', filepath)
-    return `require('${filepath}');`
-  });
+  const dependencyFolder = resolveDependencyFolder(
+    this.context.split(pathSeparator).join('/'),
+    source
+  );
 
-  return requireCalls.join(NEW_LINE);
+  return depedencies
+    .map(dependency => {
+      const trimmedDependency = isAbsolutePath(dependency)
+        ? dependency.trim()
+        : `/${dependency.trim()}`;
+      const filepath = `${dependencyFolder}${trimmedDependency}`;
+
+      if (!filepath.endsWith('/')) return `require('${filepath}');`;
+    })
+    .join(NEW_LINE);
 }

--- a/app/aem/src/server/aem-clientlib-txt-loader.js
+++ b/app/aem/src/server/aem-clientlib-txt-loader.js
@@ -1,4 +1,7 @@
+import { sep as pathSeparator } from 'path';
+
 const NEW_LINE = '\n';
+
 
 function extractBasePath(basePathExpression) {
   return basePathExpression.replace(/^\s*#base\s*=\s*/, '');
@@ -21,12 +24,13 @@ export default function aemClientLibTxtLoader(source) {
     // Ignore empty rows
     .filter(line => line);
 
-  const dependencyFolder = resolveDependencyFolder(this.context, source);
-  const requireCalls = depedencies.map(
-    dependency =>
-      `require('${dependencyFolder}${
-        isAbsolutePath(dependency) ? dependency.trim() : `/${dependency.trim()}`
-      }');`
-  );
+  const dependencyFolder = resolveDependencyFolder(this.context.split(pathSeparator).join('/'), source);
+  const requireCalls = depedencies.map(dependency => {
+    const trimmedDependency = isAbsolutePath(dependency) ? dependency.trim() : `/${dependency.trim()}`;
+    const filepath = `${dependencyFolder}${trimmedDependency}`;
+    console.log('\n[aemClientLibTxtLoader] filepath:', filepath)
+    return `require('${filepath}');`
+  });
+
   return requireCalls.join(NEW_LINE);
 }

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -2,7 +2,10 @@ import { basename, relative, sep as pathSeparator } from 'path';
 import { toJson } from 'xml2json';
 import { existsSync } from 'fs';
 
-const txtLoader = require.resolve('./aem-clientlib-txt-loader.js').split(pathSeparator).join('/');
+const txtLoader = require
+  .resolve('./aem-clientlib-txt-loader.js')
+  .split(pathSeparator)
+  .join('/');
 
 const KEY_JCR_ROOT = 'jcr:root';
 const KEY_JCR_TITLE = 'jcr:title';
@@ -13,12 +16,7 @@ const NAME_LIBS = 'libs';
 const NAME_NODE_MODULES = 'node_modules';
 
 const getRequiredHTL = (logger, component, context, pathBaseName) => {
-
   const htlFile = `${context.split(pathSeparator).join('/')}/${pathBaseName}.html`;
-  console.log('\n[getRequiredHTL] htlFile:', htlFile)
-  console.log('\n[getRequiredHTL] \`${htlFile}\`:', `${htlFile}`)
-  console.log('\n[getRequiredHTL] existsSync(htlFile):', existsSync(htlFile))
-  console.log('\n[getRequiredHTL] existsSync(\`${htlFile}\`):', existsSync(`${htlFile}`))
   if (!existsSync(`${htlFile}`)) {
     logger.info(`No HTL script for ${pathBaseName}`);
     return '';
@@ -31,21 +29,10 @@ const getRequiredClientLibs = componentDir => {
   // js.txt or css.txt files
   const loadClientLibCode = !componentDir
     ? ''
-    : `
-    var txtFileLoadContext = require.context(
-      '!!${txtLoader}!${componentDir}/', 
-      /* include subdirectories: */ 
-      true, 
-      /* all js.txt and css.txt files */
-      /(^|\\/)(js|css).txt$/
-    );
-    // Execute all files
-    txtFileLoadContext.keys().forEach(txtFileLoadContext);
-  `;
-  // : [
-  //   `var txtFileLoadContext = require.context('!!${txtLoader}!${componentDir}/', true, /(^|\\/)(js|css).txt$/);`,
-  //   `txtFileLoadContext.keys().forEach(txtFileLoadContext);`,
-  // ].join('\n');
+    : [
+      `var txtFileLoadContext = require.context('!!${txtLoader}!${componentDir}/', true, /(^|\\/)(js|css).txt$/);`,
+      `txtFileLoadContext.keys().forEach(txtFileLoadContext);`,
+    ].join('\n');
 
   return loadClientLibCode;
 };
@@ -93,12 +80,10 @@ export default async function aemComponentLoader(source) {
     component.properties[KEY_JCR_TITLE] = pathBaseName;
   }
 
-  const components = [
+  return [
     `var component = ${JSON.stringify(component)};`,
     'module.exports = component;',
     getRequiredClientLibs(context.split(pathSeparator).join('/')),
     getRequiredHTL(logger, component, context, pathBaseName),
-  ]
-  console.log('\n[aemComponentLoader] components:', components);
-  return components.join('\n');
+  ].join('\n');
 }

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -13,7 +13,7 @@ const NAME_LIBS = 'libs';
 const NAME_NODE_MODULES = 'node_modules';
 
 const getRequiredHTL = (logger, component, context, pathBaseName) => {
-  const htlFile = `${context}${sep}${pathBaseName}.html`;
+  const htlFile = `${context}${pathSeparator}${pathBaseName}.html`;
   if (!existsSync(htlFile)) {
     logger.info(`No HTL script for ${pathBaseName}`);
     return '';
@@ -50,7 +50,7 @@ const getRequiredClientLibs = componentDir => {
  * @returns {string} The resource type.
  */
 const getResourceType = (rootContext, context) => {
-  const segs = relative(rootContext, context).split(sep);
+  const segs = relative(rootContext, context).split(pathSeparator);
   // if component is in different module, it will be below a node_modules
   let idx = segs.indexOf(NAME_NODE_MODULES);
   if (idx >= 0) {
@@ -66,7 +66,7 @@ const getResourceType = (rootContext, context) => {
       segs.splice(0, 1);
     }
   }
-  return segs.join(sep);
+  return segs.join(pathSeparator);
 };
 
 export default async function aemComponentLoader(source) {

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -13,7 +13,7 @@ const NAME_LIBS = 'libs';
 const NAME_NODE_MODULES = 'node_modules';
 
 const getRequiredHTL = (logger, component, context, pathBaseName) => {
-  const htlFile = `${context}${pathSeparator}${pathBaseName}.html`;
+  const htlFile = `${context}${sep}${pathBaseName}.html`;
   if (!existsSync(htlFile)) {
     logger.info(`No HTL script for ${pathBaseName}`);
     return '';
@@ -50,7 +50,7 @@ const getRequiredClientLibs = componentDir => {
  * @returns {string} The resource type.
  */
 const getResourceType = (rootContext, context) => {
-  const segs = relative(rootContext, context).split(pathSeparator);
+  const segs = relative(rootContext, context).split(sep);
   // if component is in different module, it will be below a node_modules
   let idx = segs.indexOf(NAME_NODE_MODULES);
   if (idx >= 0) {
@@ -66,7 +66,7 @@ const getResourceType = (rootContext, context) => {
       segs.splice(0, 1);
     }
   }
-  return segs.join(pathSeparator);
+  return segs.join(sep);
 };
 
 export default async function aemComponentLoader(source) {

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -13,7 +13,7 @@ const NAME_LIBS = 'libs';
 const NAME_NODE_MODULES = 'node_modules';
 
 const getRequiredHTL = (logger, component, context, pathBaseName) => {
-  const htlFile = `${context}/${pathBaseName}.html`;
+  const htlFile = `${context}${pathSeparator}${pathBaseName}.html`;
   if (!existsSync(htlFile)) {
     logger.info(`No HTL script for ${pathBaseName}`);
     return '';
@@ -66,12 +66,11 @@ const getResourceType = (rootContext, context) => {
       segs.splice(0, 1);
     }
   }
-  return segs.join('/');
+  return segs.join(pathSeparator);
 };
 
 export default async function aemComponentLoader(source) {
   const { context, rootContext } = this;
-  context = context.replace(/\/\//g, '\/');
   const logger = this.getLogger();
   const componentData = JSON.parse(toJson(source));
   const pathBaseName = basename(context);

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -71,7 +71,7 @@ const getResourceType = (rootContext, context) => {
 
 export default async function aemComponentLoader(source) {
   const { context, rootContext } = this;
-  context = context.replaceAll('//', '/')
+  context = context.replace(new RegExp('\/\/', 'g'), '/');
   const logger = this.getLogger();
   const componentData = JSON.parse(toJson(source));
   const pathBaseName = basename(context);

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -2,7 +2,7 @@ import { basename, relative, sep as pathSeparator } from 'path';
 import { toJson } from 'xml2json';
 import { existsSync } from 'fs';
 
-const txtLoader = require.resolve('./aem-clientlib-txt-loader.js');
+const txtLoader = require.resolve('./aem-clientlib-txt-loader.js').split(pathSeparator).join('/');
 
 const KEY_JCR_ROOT = 'jcr:root';
 const KEY_JCR_TITLE = 'jcr:title';
@@ -13,8 +13,13 @@ const NAME_LIBS = 'libs';
 const NAME_NODE_MODULES = 'node_modules';
 
 const getRequiredHTL = (logger, component, context, pathBaseName) => {
-  const htlFile = `${context}${pathSeparator}${pathBaseName}.html`;
-  if (!existsSync(htlFile)) {
+
+  const htlFile = `${context.split(pathSeparator).join('/')}/${pathBaseName}.html`;
+  console.log('\n[getRequiredHTL] htlFile:', htlFile)
+  console.log('\n[getRequiredHTL] \`${htlFile}\`:', `${htlFile}`)
+  console.log('\n[getRequiredHTL] existsSync(htlFile):', existsSync(htlFile))
+  console.log('\n[getRequiredHTL] existsSync(\`${htlFile}\`):', existsSync(`${htlFile}`))
+  if (!existsSync(`${htlFile}`)) {
     logger.info(`No HTL script for ${pathBaseName}`);
     return '';
   }
@@ -37,6 +42,11 @@ const getRequiredClientLibs = componentDir => {
     // Execute all files
     txtFileLoadContext.keys().forEach(txtFileLoadContext);
   `;
+  // : [
+  //   `var txtFileLoadContext = require.context('!!${txtLoader}!${componentDir}/', true, /(^|\\/)(js|css).txt$/);`,
+  //   `txtFileLoadContext.keys().forEach(txtFileLoadContext);`,
+  // ].join('\n');
+
   return loadClientLibCode;
 };
 
@@ -66,7 +76,7 @@ const getResourceType = (rootContext, context) => {
       segs.splice(0, 1);
     }
   }
-  return segs.join(pathSeparator);
+  return segs.join('/');
 };
 
 export default async function aemComponentLoader(source) {
@@ -78,14 +88,17 @@ export default async function aemComponentLoader(source) {
     resourceType: getResourceType(rootContext, context),
     properties: componentData[KEY_JCR_ROOT] || {},
   };
+
   if (!component.properties[KEY_JCR_TITLE]) {
     component.properties[KEY_JCR_TITLE] = pathBaseName;
   }
 
-  return [
+  const components = [
     `var component = ${JSON.stringify(component)};`,
     'module.exports = component;',
-    getRequiredClientLibs(context),
+    getRequiredClientLibs(context.split(pathSeparator).join('/')),
     getRequiredHTL(logger, component, context, pathBaseName),
-  ].join('\n');
+  ]
+  console.log('\n[aemComponentLoader] components:', components);
+  return components.join('\n');
 }

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -1,8 +1,6 @@
-import { basename, relative } from 'path';
+import { basename, relative, sep as pathSeparator } from 'path';
 import { toJson } from 'xml2json';
 import { existsSync } from 'fs';
-
-const pathSeparator = require ('os').separator;
 
 const txtLoader = require.resolve('./aem-clientlib-txt-loader.js');
 
@@ -73,6 +71,7 @@ const getResourceType = (rootContext, context) => {
 
 export default async function aemComponentLoader(source) {
   const { context, rootContext } = this;
+  context = context.replaceAll('//', '/')
   const logger = this.getLogger();
   const componentData = JSON.parse(toJson(source));
   const pathBaseName = basename(context);

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -71,7 +71,7 @@ const getResourceType = (rootContext, context) => {
 
 export default async function aemComponentLoader(source) {
   const { context, rootContext } = this;
-  context = context.replace(new RegExp('\/\/', 'g'), '/');
+  context = context.replace(/\/\//g, '\/');
   const logger = this.getLogger();
   const componentData = JSON.parse(toJson(source));
   const pathBaseName = basename(context);

--- a/app/aem/src/server/aem-component-loader.js
+++ b/app/aem/src/server/aem-component-loader.js
@@ -1,6 +1,8 @@
-import { basename, relative, sep as pathSeparator } from 'path';
+import { basename, relative } from 'path';
 import { toJson } from 'xml2json';
 import { existsSync } from 'fs';
+
+const pathSeparator = require ('os').separator;
 
 const txtLoader = require.resolve('./aem-clientlib-txt-loader.js');
 

